packages/app/src/app/pages/Search/Filters/index.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import getTemplate from 'common/lib/templates';
+import { ALGOLIA_DEFAULT_INDEX } from 'common/lib/utils/config';
 
 import Filter from './Filter';
+import Sort from './Filter/Sort';
 import { Container } from './elements';
 
 function Filters() {
   return (
     <Container>
+      <Sort
```

Comment:
This is nice! Maybe we can also re-use 

![image](https://user-images.githubusercontent.com/587016/54824409-1c559480-4cab-11e9-80cd-bc641446e2c4.png)

in a later version.